replication/row_event.go

```diff
@@ -284,12 +282,23 @@ func (e *RowsEvent) Decode(data []byte) error {
 	return nil
 }
 
+func isBitSet(bitmap []byte, i int) bool {
+	return bitmap[i/8]&(1<<(uint(i)%8)) > 0
```

Comment:
I would recommend using binary operations, i.e. `i>>3` instead of `i/8` and `i&7` instead of `i%8`.